### PR TITLE
[TCVP-1471] worked on shifting date to adjust the time

### PR DESCRIPTION
**Pull request by kkadaru**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/staff-portal/src/app/components/contact-info/contact-info.component.ts [hunk 1]
```diff
@@ -260,9 +260,9 @@ export class ContactInfoComponent implements OnInit {
 
       // set violation date and time
       let tempViolationDate = new Date(this.lastUpdatedDispute.issuedDate);
+      tempViolationDate.setHours(tempViolationDate.getHours() + 7);
```
Comment (pbolduc) [inline src/frontend/staff-portal/src/app/components/contact-info/contact-info.component.ts:263]: You cant just add 7 hours. That will not work. We have time zones and daylight savings.

Comment (Sharkww) [inline src/frontend/staff-portal/src/app/components/contact-info/contact-info.component.ts:263]: Can you please help to test removing the "new Date" and this line like below
let tempViolationDate = this.lastUpdatedDispute.issuedDate;

**Review by Sharkww — CHANGES_REQUESTED** [review]

**Review by Sharkww — COMMENTED** [review]
Please also change the date time in ticket-info.component.ts.
It should be include in the same JIRA ticket
Thanks

File: src/frontend/staff-portal/src/app/components/ticket-info/ticket-info.component.ts [hunk 2]
```diff
@@ -655,10 +655,10 @@ export class TicketInfoComponent implements OnInit {
         this.form.patchValue(this.initialDisputeValues);
 
         // set violation date and time
-        let violationDate = new Date(response.issuedDate);
+        let violationDate = response.issuedDate.split("T");
         this.form.get('violationTicket').get('issuedDate').setValue(response.issuedDate);
-        this.form.get('violationTicket').get('violationDate').setValue(this.datePipe.transform(violationDate, "yyyy-MM-dd"));
-        this.form.get('violationTicket').get('violationTime').setValue(this.datePipe.transform(violationDate, "hhmm"));
+        this.form.get('violationTicket').get('violationDate').setValue(violationDate[0]);
+        this.form.get('violationTicket').get('violationTime').setValue(violationDate[1].split(":")[0] + ":" + violationDate[1].split(":")[1]);
```
Comment (Sharkww) [inline src/frontend/staff-portal/src/app/components/ticket-info/ticket-info.component.ts:661]: Sorry that violationTime should be violationDate[1].split(".")[0].replace(":","")

.NET DateTime string is 2022-06-12T09:12:34.567

Please also correct the contact-info one and test it before commit. Thanks!

**Review by Sharkww — CHANGES_REQUESTED** [review]

**Review by Sharkww — APPROVED** [review]
@lfdgov we may need some discussion for the datetime

**Pull request merged**